modules/tools/velero/charts/velero-schedule/templates/harbor-daily-backup.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: velero.io/v1
+kind: Schedule
+metadata:
+  name: harbor-daily-backup
```

Comment:
does it make sense to include this within the the `velero` module, or should it be included within the `harbor` module if `enable_velero` is set to `true`?  I think this chart shouldn't be installed if `enable_velero` is `true` and `enable_harbor` is false.

Reply:
Added a list to control what namespaces Velero is used on. I added a feature flag in the Velero module that uses contains on this list to decide whether the Helm release for the chart containing the backup schedules for that namespace is installed. The Helm release per namespace contains one or more instances of “Schedule” API type, specific to that namespace. So we can still have different label targets, pre-/post-hooks, etc…for workloads in the namespace. Not perfect, but OK for now since we are only interested in two namespaces. Open to other ideas.